Cython/Includes/cpython/time.pxd

```diff
@@ -2,11 +2,8 @@
 Cython implementation of (parts of) the standard library time module.
 """
 
-from libc.stdint cimport int64_t
-
-ctypedef int64_t _PyTime_t
-
 cdef extern from "pytime.h":
+    ctypedef _PyTime_t  # alias for int64_t
```

Comment:
This declares it as `object`. :)

You still want to use `int64_t` here to tell Cython what it is, it's just that the `cdef extern` block lets Cython know that the actual declaration is in the header file and not here, and that it should not generate its own C declaration for it.